index.js

```diff
@@ -219,6 +223,110 @@ let ContainerService = {
     });
   },
 
+  getTabs(args) {
+    return new Promise((resolve, reject) => {
+      if (!("userContextId" in args)) {
+        reject(null);
```

Comment:
Need a reason error message here. e.g., `reject("getTabs must be called with userContextId argument.")`